### DISCUSSION TITLE
Change code content via toggle

### DISCUSSION THREAD

**Discussion by MAGICCC** [squidfunk/mkdocs-material#4757] [Q&A] [answered]
Hello,

Is it possible - like the [Color palette toggle](https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#color-palette-toggle) - to change text inside a code tag?

Let me explain my scenario:
As you know there's `docker-compose` and the `docker compose` commands. Due to that some users still use either docker-compose or the new docker compose command, but we changed our docs to docker compose. We included already the [Announcement bar](https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-header/?h=announce#announcement-bar), but users tend to not read that.

Thanks for any answer

**Comment by squidfunk** [comment, marked as answer]
Definitely possible, but I just recommend using [linked content tabs](https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#linked-content-tabs) together with content tabs to [group related code blocks](https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#grouping-code-blocks). Something like:

```
=== "New"

    ```
    docker compose ...
    ```

=== "Old"

    ```
    docker-compose ...
    ```

```

You can of course adjust the contents/titles so they fit your taste.

**Reply by MAGICCC** [reply]
Thanks, that will do the trick!

**Reply by DerLinkman** [reply]
Awesome! Is there a way to hide that switch? For example (their all toggled by the same click):

![grafik](https://user-images.githubusercontent.com/62480600/207873570-7bb4d76b-a4db-472f-9602-86a7834b6fc2.png)

It´s showed multiple times in a row. Maybe a variable behind the switch name like "hidden=true" would be cool. Or is there a build in way to do this?

**Reply by squidfunk** [reply]
Nope, we currently can't add attributes to content tabs to make them discernable. Related:

- https://github.com/squidfunk/mkdocs-material/issues/4268
